src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -75,13 +187,7 @@ export CARGO_TARGET_X86_64_FUCHSIA_RUSTFLAGS="-C link-arg=--sysroot=${SDK_PATH}/
 These can be run together in a shell environment by executing
 `(source config-env.sh && ./x.py install)`.
 
-## Building Rust programs
-
-After compiling Rust binaries, you'll need to build a component, package it, and
-serve it to a Fuchsia device or emulator. All of this can be done using the
-Fuchsia SDK.
-
-As an example, we'll compile and run this simple program on a Fuchsia emulator:
+Once `rustc` is installed, we can now create this simple program on a Fuchsia emulator:
```

Comment:
Suggest reword: "Once `rustc` is installed, we can compile this basic program for Fuchsia:".